.gitignore

```diff
@@ -23,3 +23,7 @@
 
 # Ignore ReadTheDocs site style
 /docs/site
+
+# Ignore vendor / bundle
+/.bundle/
+/vendor/bundle
```

Comment:
As mentioned in #139, please remove from this change from the PR. Should fix the merge conflict issues and be a cleaner merge.